### PR TITLE
Project setup

### PR DESCRIPTION
**Pull request by HafizMassamTabraizKhan**
In this branch, I did the following:

Set up a React app.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,211 @@
...
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://react.dev/">HTML</a></li>
+    <li><a href="https://html.com/">HTML</a></li>
+    <li><a href="http://www.css.com/">CSS</a></li>
+    <li><a href="http://www.javascript.com/">JavaScript</a></li>
+    <li><a href="https://webpack.js.org/">Webpack</a></li>
+  </ul>
```
Comment (Gambit142) [inline README.md]: - [ ] Some of the tech stacks listed here were not used in this project. Kindly remove them. 👍

File: README.md [hunk 2]
```diff
@@ -1 +1,211 @@
...
+2. Display result.
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="https://github.com/HafizMassamTabraizKhan/Math-Magicians"></a>
+
+
+- [[Live Demo Link](https://github.com/HafizMassamTabraizKhan/Math-Magicians)]
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```
Comment (Gambit142) [inline README.md]: - [ ] Kindly remove the Live Demo section of the README.md file. The Live Demo section should be in your README if you deployed the project. Kindly remove it. 👍

**Review by Gambit142 — COMMENTED** [review]
Hi @HafizMassamTabraizKhan 👋, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

- Kindly check the comments below for invalidation reasons. 👍 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Review by binhussen — APPROVED** [review]
Hi @HafizMassamTabraizKhan , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

## Highlights
- Created with Create React App ✔️
- Added react-scripts✔️
- Used Git flow ✔️
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**